### DISCUSSION TITLE
Manually restart the CRON jobs

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15967] [How to] [answered]
Hi,

I using WAMP server to use the Yetiforce and it was running normally for past one month. 

Today the cron job for the wrok flow was continuously running. 
![image](https://user-images.githubusercontent.com/96602176/164170569-1eb5e0f8-7d5c-4ecb-a1b4-49c2d763f33d.png)

Even after restarting the WAMP server and PC, still the CRON job status remains same in the workflow. 

Could you please guide me to restart the CRON jobs to run it for all the jobs. 

with regards,
Rajasekaran.

**Comment by rskrzypczak** [comment, marked as answer]
Probably one of the workflow actions is causing an error. You can try to debug it or disable worflow actions one by one and watch when the problem goes away.

**Reply by Ilamparithi-Tamizhanangu** [reply]
Thank you.

I deleted the workflow created in the last session and now its working fine. 

Thanks again for your support
